src/middleware/cache/index.ts

```diff
@@ -5,28 +5,60 @@ export const cache = (options: {
   cacheName: string
   wait?: boolean
   cacheControl?: string
+  vary?: string
```

Comment:
How about accepting `string[]`?

```ts
vary: ['Vary', 'Accept-Encoding']
```

I think it makes better DX.

Reply:
Nice idea. Thank you very much.

[23e339e](https://github.com/honojs/hono/pull/2426/commits/23e339e0e8ec9dab7446a3ad3806b4c29ffc05a9)

Reply:
@naporin0624 

Sorry for not fully explaining! I think `string | string[]` is best!

Reply:
You meant to accept string[] as well! It would be even better to accept the original string, but also an array.

Thanks for pointing that out.